### ISSUE TITLE
Collecting Awesome Usage Of {ffscrapr}

### ISSUE THREAD

**Issue by tanho63** [dynastyprocess/ffscrapr#280]
Got any cool examples of how you've used ffscrapr? 

I'd love to hear about it and build up a directory of blog posts, code snippets, Shiny apps etc that can help inspire future analysis and dev! Please comment here if you've got such a thing.

Examples of things I'm keen on hearing about:
- managing a salary cap league
- tracking an MFL auction
- revisiting league history and bad beats
- and anything else!

**Comment by TheMathNinja** [comment]
Things I've created via ffscrapr:

1. Tracked historical value of rookie draft pick by draft slot in our dynasty league (using ff_draft() in conjunction with using ff_playerscores() for points and ff_starters() / ff_starterpositions() to calculate positional baselines)
2. Tracked salary allocation vs. player value to determine positional buy-lows / auction targets in a salary cap league
3. Used ff_franchises() in conjunction with ff_standings() to predict team performance before Week 1 and set equitable schedules accordingly.

**Comment by tanho63** [comment]
@TheMathNinja Awesome! Any code examples published anywhere? 😀

**Comment by TheMathNinja** [comment]
Yes, the scheduler is public (the other ones would take a bit of work for me to take out my "special sauce"). 

Just be forewarned that this code was written almost a year ago, so it's a much earlier version of ffscrapr, and also just 1-2 months into me learning how to code in R, haha. I'll re-visit it this July/August for an update, I imagine.

https://github.com/TheMathNinja/NFLdataprojects/blob/master/Football-Analytics-Labs/FAFLschedgeneratorv3.R

**Comment by draftydesert277** [comment]
I don't know if you're familiar with Tableau, but you could check out the beginnings of a "league dashboard" I'm working on: https://public.tableau.com/profile/steve.goedken#!/vizhome/Book1_16193733190810/Records

**Comment by cincysam6** [comment]
Tan,

This is a really great package. Working on a Shiny App to compile a league website that will look at a wide variety of historical league stats that ESPN doesn't keep like all time record vs opponent. The league goes back on ESPN to like 2007 and has had stable members the whole time so really interested in seeing who historically the best drafter in the league has been, if any owner is particularly adept at late round value, things like that.

**Comment by cincysam6** [comment]
@tanho63 Building a Blog on my fantasy league. Eventually will do some more sophisticated things and want to build a shiny app, but this has been fun for my family fantasy league that we've done for 25 years. 

https://kfl-analytics.netlify.app/

**Comment by TheMathNinja** [comment]
Another great use: IDP commissioners know how controversial position designation can be. I use mfl_players() and espn(players) to scrape for position discrepancies between the 2 platforms and then use this short list to let PFF break ties. Saves so much time and controversy as a commissioner!
